src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/ServerConfService.java

```diff
@@ -62,6 +63,22 @@ public ServerConfType getServerConf() {
         return serverConfRepository.getServerConf();
     }
 
+    /**
+     * Get a server conf; an existing server conf will be returned if one exists. Otherwise
+     * a new transient instance is returned.
+     * @return
+     */
+    public ServerConfType getOrCreateServerConf() {
+        ServerConfType serverConfType = null;
+        try {
+            serverConfType = getServerConf();
+        } catch (CodedException ce) {
+            // server conf doesn't exist which is fine - let's just create one
```

Comment:
Same comment as in `isServerConfInitialized` about CodedExceptions.

Reply:
Fixed